data/current/metadata.json

```diff
@@ -1,445 +1 @@
vision_id": "4c70bab9-7094-4320-ae74-766783db6f02", "data_quality": []}, {"cache_last_updated": null, "unique_identifier": "", "package_id": "dfa4daf1-669e-4514-82cd-982f27707ed0", "datastore_active": false, "id": "807c638f-10e7-4c79-b756-677d3af7265c", "character_set": "", "state": "active", "name_translated": {"fr": "Table 5 - Allocation au survivant", "en": "Table 5 - Allowance for the Survivor"}, "hash": "", "description": "", "format": "XLS", "date_published": "2015-12-01", "mimetype_inner": null, "url_type": "", "mimetype": null, "cache_url": null, "name": "Table 5 - Allowance for the Survivor", "language": ["en", "fr"], "created": "2016-12-29T12:30:17.927907", "url": "http://www.edsc-esdc.gc.ca/ouvert-open/isp/oas_gis-psv_srg/Table5_Allowance_for_the_survivor_January2020.xlsx", "last_modified": null, "resource_type": "dataset", "position": 9, "revision_id": "4c70bab9-7094-4320-ae74-766783db6f02", "data_quality": []}], "place_of_publication": [], "metadata_contact": {}, "num_resources": 10, "topic_category": [], "tags": [], "spatial": "", "collection": "primary", "org_title_at_publication": {"fr": "Emploi et D\u00e9veloppement social Canada", "en": "Employment and Social Development Canada"}, "date_published": "2015-12-01 00:00:00", "display_flags": [], "groups": [], "creator_user_id": "aa584ab4-544c-4c5c-81da-d1cff9bd96fa", "data_series_name": {"fr": "", "en": ""}, "relationships_as_subject": [], "license_title": "Open Government Licence - Canada", "time_period_coverage_end": "2020-03-31 00:00:00", "restrictions": "unrestricted", "organization": {"description": "", "created": "2016-09-23T17:25:26.246118", "title": "Employment and Social Development Canada | Emploi et D\u00e9veloppement social Canada", "name": "esdc-edsc", "is_organization": true, "state": "active", "image_url": "", "revision_id": "a1b880bc-80de-4885-8790-a8453a7a346e", "type": "organization", "id": "055ADCD8-8FDA-40DA-A5B7-689FF7DB57A6", "approval_status": "approved"}, "name": "dfa4daf1-669e-4514-82cd-982f27707ed0", "isopen": false, "license_id": "ca-ogl-lgo", "url": "{u'fr': u'', u'en': u'http://www.esdc.gc.ca/en/cpp/oas/index.page?'}", "notes": "This dataset provides information on Benefits Amounts for Income Supplement and the Allowances according to income level and marital status. This is updated on a quarterly basis. The following tables of amounts will provide you with the amount of your monthly benefit, which will be based on your age, income level and marital status. PDF and html version are available on other web page: \r\nhttps://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security/payments.html#amounts\r\n\r\n", "owner_org": "055ADCD8-8FDA-40DA-A5B7-689FF7DB57A6", "audience": [], "license_url": "http://open.canada.ca/en/open-government-licence-canada", "program_page_url": {"fr": "", "en": ""}, "title": "Old Age Security (OAS) - Table of Benefit Amounts by marital status and income level", "revision_id": "9d44d6e4-7512-4765-9647-64ccdf60c546"}}
```

Comment:
Just to make it easier for history to see what was changed, can you run this through json prettify so that it's on multiple lines, see [this previous data update PR](https://github.com/danielabar/gis-lookup/pull/5/files) for comparison.

(I should update the maintenance section in readme about this)

Reply:
Ran prettier